### ISSUE TITLE
Time Zone support

### ISSUE THREAD

**Issue by avigilante** [davish/obsidian-full-calendar#252]
**Describe the bug**
[I absolutely love FullCalendar!!!]
I publish my MSFT Office365 calendar as iCal. I subscribe to it from my Google account.
I receive meetings in different time zones. They all show at the correct time in Office365 and Google.
However, FullCalendar displays the meetings from other time zones as the time was set in my time zone: i.e. a meeting set for 10:00a US Eastern Time shows as 10:00a US Pacific Time in my FullCalendar (as opposed to 7:00a US Pacific Time).
Meetings originally set in my time zone show correctly.

Basically, FullTime seems to ignore the time zone part of the meeting time, and only considered the numeral part of the time.
Or maybe there’s a way to set my time zone in FullCalendar and I can’t find it?

**Comment by davish** [comment]
Thanks for submitting and for the kind words! This is issue is already tracked in #66.
